### PR TITLE
Add auto-evo prediction calculating icon

### LINKED ISSUE #2768: Create a custom icon for auto-evo prediction being calculated
I reused the help button question mark but it should get its own icon.

### PR DESCRIPTION
**Pull request by Kasterisk**
**Brief Description of What This PR Does**

**Related Issues**

Closes #2768.

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: assets/textures/gui/bevel/autoevo.png.import [hunk 1]
```diff
@@ -0,0 +1,35 @@
+[remap]
```
Comment (hhyyrylainen) [inline assets/textures/gui/bevel/autoevo.png.import:1]: I think the icon file should be named something like "autoEvoPending.png" to be more clear what it is (it's not a general icon used to refer to auto-evo).

**Comment by hhyyrylainen** [comment]
The icons displayed on the values that have the up/down arrows are now much larger. The arrows appear as blurry on my screen as a result... I think something needs to be done to detach the size of the pending icon from the up/down arrow sizes.

**Comment by Kasterisk** [comment]
Somehow missed that 🤦‍♂️

**Comment by hhyyrylainen** [comment]
Should the pending icon have a separate size or does your latest commit handle that already?

**Comment by Kasterisk** [comment]
Yep, I added another exported property handling the size. https://github.com/Revolutionary-Games/Thrive/blob/3996eedb205e8596fa9a573b6e23f0fec9e58c57/src/microbe_stage/editor/CellEditorComponent.tscn#L1164

**Review by hhyyrylainen — APPROVED** [review]

**Pull request merged**